packages/scribe/src/components/NodesMenu/Menu/MenuContext.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react-refresh/only-export-components */
```

Comment:
FYI [This issue](https://github.com/ArnaudBarre/eslint-plugin-react-refresh/issues/25#issuecomment-1729071347) seems to indicate this might be a real issue (with a valid solution) and ignoring the linter might not be a good idea - unless fast-refresh is working fine for you?